jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -485,7 +483,7 @@ public void resize(int width, int height, boolean fixAspect) {
     }
 
     /**
-     * <code>getFrustumBottom</code> returns the value of the bottom frustum
+     * Returns the value of the bottom frustum
```

Comment:
Completely aside from the format/styling concerns, I question the value of this, and other "Trivial" Javadoc comments in this file. It adds no details that cannot be gleaned from the method signature, and therefore does not add value above what the javadoc tool would create on it's own. Why does it exist?